examples/kitchen-sink-example/next.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   poweredByHeader: false,
-  target: 'serverless'
+  target: 'serverless',
+  env: {
+    AUTH0_BASE_URL: process.env.VERCEL_URL || process.env.AUTH0_BASE_URL
```

Comment:
Wouldnt it make more sense to flip the order here? 

```
AUTH0_BASE_URL: process.env.AUTH0_BASE_URL || process.env.VERCEL_URL
```

In that case we set the `AUTH0_BASE_URL` to `VERCEL_URL` unless `AUTH0_BASE_URL` was explicitly set.

Reply:
I don't think it makes a difference. On Vercel `AUTH0_BASE_URL` wont be set and in development `VERCEL_URL` wont be set.

I'd rather prefer the production configuration on the rare occaison where both might be set.